prover/src/lib.rs

```diff
@@ -37,6 +38,8 @@ impl ProverInput {
         previous: Option<PlonkyProofIntern>,
         save_inputs_on_error: bool,
     ) -> Result<GeneratedBlockProof> {
+        use anyhow::Context as _;
```

Comment:
I am curios, any deeper reason to use ` as _` often through the code when importing types?

Reply:
It's my preference, especially for extension traits, as the trait will never need to be named in e.g a bound.

Reply:
Wouldn't that introduce confusion for those less versed in Rust?

Reply:
I don't think it's a big cognitive leap, especially since most rust devs are familiar with `let _ = ...`. `rust-analyzer` has even started explicitly suggesting this pattern when you use an extension method.